link/src/test/java/com/stripe/android/link/ui/inline/InlineSignupViewModelTest.kt

```diff
@@ -115,31 +115,6 @@ class InlineSignupViewModelTest {
             assertThat(viewModel.signUpState.value).isEqualTo(SignUpState.InputtingPhoneOrName)
         }
 
-    @Test
```

Comment:
Why was this test removed?

Reply:
I thought I added a comment here about that 🤔

I removed it in favor of the `User input is valid without name for US users` and `User input is only valid with name for non-US users` tests further down in this file.